common/app/model/EmailAddons.scala

```diff
@@ -321,6 +321,11 @@ case object WinterOlympics2018 extends FrontEmailMetadata {
   override val banner = Some("winter-olympics-2018.png")
 }
 
+case object WinterOlympics2022 extends FrontEmailMetadata {
+  val name = "Winter Olympics 2022"
+  override val banner = Some("winter-olympics-2022.png")
+}
```

Comment:
Does this object need to be included in the `frontEmails` sequence at the bottom of the file?

Reply:
Yes, thanks. I've added that